studio/db_providers/db_provider_setup.py

```diff
@@ -0,0 +1,115 @@
+"""Data providers."""
+import os
+import yaml
```

Comment:
This module functionally is part of former model_setup.py with logic related to experiment "database" separated out.